src/applications/personalization/profile360/components/REMOVAL_PROFILE_NOTES

```diff
@@ -0,0 +1,37 @@
+
+## Components also used in profile 2
+src/applications/personalization/profile360/components/BankInfoForm.jsx
+src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx
+src/applications/personalization/profile360/components/DowntimeBanner.jsx
+src/applications/personalization/profile360/components/LoadFail.jsx
+src/applications/personalization/profile360/components/PaymentInformationBlocked.jsx
+src/platform/user/profile/vet360/components/base/Vet360EditModalErrorMessage.jsx
+src/platform/user/profile/vet360/components/PhoneField/PhoneEditView.jsx
+src/platform/user/profile/vet360/components/AddressField/AddressEditView.jsx
+src/platform/user/profile/vet360/components/AddressField/AddressField.jsx
+
+src/platform/user/profile/vet360/components/base/Vet360Transaction.jsx
+src/platform/user/profile/vet360/components/base/Vet360TransactionPending.jsx
+src/platform/user/profile/vet360/components/base/Vet360TransactionInlineErrorMessage.jsx
+src/platform/user/profile/vet360/components/base/Vet360TransactionErrorBanner.jsx
+
+
+
+## Other reused components
+src/applications/personalization/profile360/components/ContactInformation.jsx USED BY src/applications/gi/components/vet-tec/VetTecInstitutionProfile.jsx
+src/applications/personalization/profile360/components/ContactInformationExplanation.jsx USED BY src/applications/gi/components/vet-tec/VetTecInstitutionProfile.jsx
+src/platform/user/profile/vet360/components/ResidentialAddress.jsx USED BY src/applications/gi/components/vet-tec/VetTecInstitutionProfile.jsx
+src/platform/user/profile/vet360/components/FaxNumber.jsx USED BY src/applications/gi/components/vet-tec/VetTecInstitutionProfile.jsx
+src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx USED BY src/applications/letters/letters-entry.jsx
+src/platform/user/profile/vet360/components/MailingAddress.jsx USED BY src/applications/letters/letters-entry.jsx
```

Comment:
I think we should likely keep all the standalone Profile 1 address components around just in case someone needs to be able to easily drop in a piece of Veteran contact info.

Reply:
Sounds good, I have kept all of the address components.